plugins/topology.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: topology
+spec:
+  version: v0.0.1
+  shortDescription: Provides insight into the topology of a Kubernetes cluster
```

Comment:
Better description for you: 

> Browse zones/regions of Nodes or Pods

Reply:
> Explore region topology for nodes or pods

Reply:
I appreciate the suggestion, but I have this same description in the `README.md` file for now. Would you mind me changing that later on?

Reply:
I think the question would be why not now? :)
You can have different descriptions in different places.

Reply:
Yeah, guess you're right. I've amended the description as per one of your suggestions.